protobuf/README.md

```diff
@@ -8,12 +8,12 @@ Check out the [Flow Access API specification](/docs/content/access-api-spec.md).
 
 ## Generating Go stubs
 
-You can use [prototool](https://github.com/uber/prototool) to generate gRPC client stubs in a variety of languages.
+You can use [buf](https://github.com/bufbuild/buf) to generate gRPC client stubs in a variety of languages.
 
-Running the command below (in the current directory) will generate stubs for Go:
+Running the command below (in the root directory) will generate stubs for Go:
 
```

Comment:
Prerequisite: Install [buf](https://docs.buf.build/installation/)

Reply:
Yes, this is reflected in the README updates.